EC-84: Revise Configuration, Customers, eContent, Login and Logging (#5)
ezsystems/commerce-documentation: docs/guide/econtent/econtent_api.md

# eContent API

> eContent provides a flexible way to define project specific attributes for products and categories (and other product-related types of data).

Reviewer: ```suggestion
eContent provides a flexible way to define project-specific attributes for products and categories (and other product-related types of data).
```

eZ Commerce requires that some of the attributes use predefined identifiers.

## Categories

|Identifier|Type|Description|Standard mapping|Stored in|
|--- |--- |--- |--- |--- |
|`ses_name`|ezstring|The name of the category|`catalogElement.name`|data_text|

> |`code`|ezstring|A unique id for the category|||

Reviewer 2: ```suggestion
|`code`|ezstring|A unique ID for the category|||
```

|`text`|ezstring|Longer text for the category|`catalogElement.text`|data_text|

## Products

|Identifier|Type|Description|Standard mapping|Stored in|
|--- |--- |--- |--- |--- |
|`ses_name`|ezstring|The name of the product|catalogElement.name|data_text|
|`ses_sku`|ezstring|The unique SKU|catalogElement.sku|data_text|

> |`ses_subtitle`|ezstring|The subtitle used in product listing pages|catalogElement.text and catalogElement.subtitle</br>catalogElement.text will be replaced by the new attribute subtitle in the future|data_text|

Reviewer: ```suggestion
|`ses_subtitle`|ezstring|The subtitle used in product listing pages|catalogElement.text and catalogElement.subtitle</br>catalogElement.text will be replaced by the new `subtitle` attribute in the future|data_text|
```

|`ses_short_description`|ezstring|A short description|catalogElement.shortDescription|data_text|
|`ses_long_description`|ezstring||catalogElement.longDescription|data_text|

> |`ses_unit_price`|ezprice|The list prices for the product based on the default currency|catalogElement.price|data_float|

Reviewer: ```suggestion
|`ses_unit_price`|ezprice|A list price for the product in the default currency|catalogElement.price|data_float|
```

> |`ses_unit_price`|ezprice|The list prices for the product based on the default currency|catalogElement.price|data_float|

Reviewer 2: ```suggestion
|`ses_unit_price`|ezprice|A list of prices for the product in the default currency|catalogElement.price|data_float|
```

|`ses_vat_code`|ezstring||catalogElement.vatCode|data_text||
|`ses_unit`|ezstring|The unit of the product|catalogElement.unit|data_text|

> |`ses_image_main|ezstring|Relative path to the main image|catalogElement.mainImage|data_text|

Reviewer: ```suggestion
|`ses_image_main|ezstring|A relative path to the main image|catalogElement.mainImage|data_text|
```

> |`ses_specification`|ezstring|A JSON-formated set of data containing grouped specifications|catalogElement.specifications|data_text|

Reviewer: ```suggestion
|`ses_specification`|ezstring|A JSON-formatted set of data that contains grouped specifications|catalogElement.specifications|data_text|
```

Data for variant products:

| Identifier  | Type |Description     | Standard mapping   | Stored in  |
| -------- | -------- | ---------- | ------------------- | ---------- |

> | `ses_variants` | ezstring | Contains the SKU/variantcode, name, list price and characteristic for the variants | will be converted in order to build an variantProductNode | data\_text |

Reviewer: ```suggestion
| `ses_variants` | ezstring | Contains the SKU/variant code, name, list price and characteristic for the variants | Will be converted to build a variantProductNode | data\_text |
```
